platforms/shared/images/alpine-utils.Dockerfile

```diff
@@ -2,4 +2,5 @@ FROM alpine
 
 WORKDIR /usr/src
 
-RUN apk add --no-cache curl jq
\ No newline at end of file
+RUN apk add --no-cache curl jq wget openssl
```

Comment:
Please check if any of these is non-distributable, we should understand the reason for not creating a container image with these packages in it.